mw-webapp/src/component/table/columns.tsx

```diff
@@ -22,14 +22,13 @@ const getObjectArrayItem = (arrayItem: JobDone | PlanForNextPeriod | CurrentProb
   );
 };
 
-const getStringArrayItem = (arrayItem: string, index: string) => {
+const getStringArrayItem = (arrayItem: string, index: string, isDone?:boolean) => {
```

Comment:
1. weird function name, rename it or at least leave comment please) I see it not youth but let's make our code better) 
2. usually if want to declare function with few parameters it is a good idea to use object as singular parameter. In this case your parameters become named object fields, you will be able to create a model for your parameters. Also It is pretty scalable solution.
3. if some function return jsx - let's name it with prefix `render*`. Could you add it to conventions, please?

Reply:
1. Problem is, this function is used by: stunderComments, mentorComments and learnedForToday, i can't really think of the more specific name for it. Maybe just "renderStringItem"

2. Considering the above-mentioned, if i understood correctly, i would need to create StringItem only in Business models?

3. Will add to conventions.

Reply:
1. Provide appropriate comment for it
2. No, I mean signature of this function itself, instead of 3 parameters use only one - object
3. Super) 👍